src/core/services/AuthService.ts

```diff
@@ -30,6 +33,8 @@ class AuthService {
       // Otherwise use logged in user
       return request.user;
     }
+
+    return undefined; // Not authenticated
```

Comment:
Is this not unreachable code?

Reply:
You are right, thank you

Reply:
Is this intentional or a bug?

Reply:
I will restructure the code a bit to make this easier to read.